bindings/go/src/fdb/fdb.go

```diff
@@ -345,7 +345,7 @@ func createDatabase(clusterFile string) (Database, error) {
 	db := &database{outdb}
 	runtime.SetFinalizer(db, (*database).destroy)
 
-	return Database{clusterFile, true, db}, nil
+	return Database{clusterFile: clusterFile, isCached: true, database: db}, nil
```

Comment:
This is strictly unnecessary to fix the issue at hand but I found it helps in my IDE when looking for references of the fields.